chat-widget/src/components/livechatwidget/common/reconnectChatHelper.ts

```diff
@@ -21,7 +21,7 @@ import { TelemetryHelper } from "../../../common/telemetry/TelemetryHelper";
 // Return value: should start normal chat flow when reconnect is enabled
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const handleChatReconnect = async (chatSDK: any, props: any, dispatch: Dispatch<ILiveChatWidgetAction>, setAdapter: any, initStartChat: any, state: ILiveChatWidgetContext): Promise<boolean> => {
-    if (!isReconnectEnabled(props.chatConfig) || isPersistentEnabled(props.chatConfig)) return false;
+    if (!isReconnectEnabled(props.chatConfig) || isPersistentEnabled(props.chatConfig)) return true;
```

Comment:
Were we leveraging the old `false` return value anywhere at all?

Reply:
It was previously protected by other logic 
![image](https://github.com/microsoft/omnichannel-chat-widget/assets/14852927/e5295e3e-4299-4314-a8b7-a37608e0b4fe)

Reply:
Moved the check outside as suggested